badges/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Badges</title>
+    <link rel="stylesheet" href="badge.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+
+<body>
+
+    <nav class="navigation-container">
+        <div class="nav-header"> <a class="nav-header-item" href="/">Pro<span class="heading-inverted">CODE<span
+                        class="logo-version"> v1.0</span></a>
+        </div>
+        <div class="nav-item"> <a class="nav-list-item" href="/">DOCUMENTATION</a> </div>
+    </nav>
+
+    <div class="main-container">
+
+        <header>
+            <h3>Badge</h3>
+            <p>Badges are small circles, positioned either at top-right or bottom-right of the parent component. It can
+                be used to display numbers and status depending on where they are used.</p>
+        </header>
+        <hr>
+    </div>
+
+    <section class="icon-badges">
+
+        <p class="badge-heading">Badge on Icons</p>
+
+        <div class="icons-container">
+
+            <div class="badge-element">
```

Comment:
I will suggest wrapping badge and the element on which  you want to place badge by a div 
which have position: relative and set the height and width to fit-content